sparkcore/src/main/scala/quasar/physical/sparkcore/fs/hdfs/package.scala

```
@@ -175,7 +175,7 @@ package object hdfs {
     def fileSystem: Task[HdfsFileSystem] = Task.delay {
       val conf = new Configuration()
       conf.setBoolean("fs.hdfs.impl.disable.cache", true)
-      HdfsFileSystem.get(new URI(sparkFsConf.hdfsUriStr), conf)
```

I should clarify, either a test or confidence that it won't fail silently is sufficient.